examples/wallets-demo/src/App.tsx

```diff
@@ -29,15 +30,17 @@ export function App() {
       }
       setLoading(false);
     };
-    getAllBlockchains();
+    getAllBlockchains().catch((error) => {
```

Comment:
use `void getAllBlockchains()` to solve the eslint.

Reply:
Done